public/app/plugins/panel/geomap/GeomapPanel.tsx

```diff
@@ -436,7 +436,7 @@ export class GeomapPanel extends Component<Props, State> {
       });
     }
 
-    const found = layers.length ? true : false;
+    const found = !!layers.length;
```

Comment:
slight preference for:
```suggestion
    const found = Boolean(layers.length);
```